src/prefect/orion/schemas/states.py

```diff
@@ -46,6 +46,7 @@ class StateDetails(PrefectBaseModel):
     scheduled_time: DateTimeTZ = None
     cache_key: str = None
     cache_expiration: DateTimeTZ = None
+    returns_untrackable_result: bool = False
```

Comment:
I have a preference for `untrackable_result`. The state does not "return" the result, it's assigned to it. We happen to use the return value of the user's function most of the time, but we also manually set this data to other things like exceptions.